pkg/base1/cockpit.js

```diff
@@ -2763,19 +2763,22 @@ function full_scope(cockpit, $, po) {
             throw "invalid date or offset";
     }
 
-    function MetricsChannel(interval, options) {
+    function MetricsChannel(interval, orig_options) {
```

Comment:
I would prefer if fallbacks were implemented as ```options, ...```. In other words you specify N dicts of options, and we fallback to the next one if the first one is ```not-supported```